docs/Python-Intro.rst

```diff
@@ -14,13 +14,7 @@ This document gives a basic walkthrough of LightGBM Python-package.
 Install
 -------
 
-You can install LightGBM via pip
-
-::
-
-    pip install lightgbm
-
-Refer to `Python-package`_ folder for the detailed installation guide.
+Refer to `Python-package`_ folder for the installation guide.
```

Comment:
@raybellwaves please revert this. We still want to make it clear that `pip install` is the preferred installation method, since we maintain the packages on PyPi.

I'd support a change where `You can install LightGBM via pip` becomes `The preferred way to install LightGBM is via pip`.